io.openems.edge.predictor.similardaymodel/readme.adoc

```diff
@@ -0,0 +1,3 @@
+= io.openems.edge.predictor.similardaymodel
+
```

Comment:
Fix title + add some descriptive information like "Similar day model uses the same weekday of the last 4 weeks - where 4 is configurable - to predict the power" ... "This approach is specifically useful to predict Consumption." "Accuracy of this approach has been scientifically verified within the EMSIG project https://openems.io/research/emsig/"...